src/GraphQLParser/ParserContext.Parse.cs

```diff
@@ -524,31 +524,31 @@ private GraphQLField ParseField()
         // start of alias (if exists) equals start of field
         int start = _currentToken.Start;
 
-        var nameOrAliasComment = GetComment();
+        var comment = GetComment(); // Greedy parsing for comment here - we read comment for field itself, not for alias.
```

Comment:
@Shane32 I changed commmentss parsing for `Field`. Now it corresponds to the rest of the code when the comment refers to the object that is parsing by this method. I also changed tests.